literature.bib

```diff
@@ -5147,6 +5163,26 @@ @article{DBW:SCP18
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2017 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{SKFD:GCPR17,
+  author       = {Alexander Schulthei{\ss} and
+                  Christoph K{\"{a}}ding and
+                  Alexander Freytag and
+                  Joachim Denzler},
+  editor       = {Volker Roth and
+                  Thomas Vetter},
+  title        = {Finding the Unknown: Novelty Detection with Extreme Value Signatures
+                  of Deep Neural Activations},
+  booktitle    = GCPR,
+  series       = LNCS,
+  volume       = {10496},
+  pages        = {226--238},
+  publisher    = Springer,
+  year         = 2017,
+  pdf          = {https://pub.inf-cv.uni-jena.de/pdf/Schultheiss17_FTU.pdf},
+  url          = {https://doi.org/10.1007/978-3-319-66709-6\_19},
+  doi          = {10.1007/978-3-319-66709-6\_19},
+}
+
```

Comment:
Field `address` is missing for this entry. Maybe Cham? Please check or otherwise report that you could not find it (see Readme on branch `sk` for information on how to do that).